x/auth/exported/exported.go

```diff
@@ -1,62 +0,0 @@
-package exported
-
-import (
```

Comment:
@alexanderbez are we removing the exported folder? Don't believe this was intended?

Reply:
as long as it doesn't create dep cycles it's ok with me removing it

Reply:
I think it was intended. As long as we can avoid import cycles, I think this is fine.

From the spec:

```
However, these
contracts can define methods that operate on and/or return types that are specific
to the contract's implementing module and this is where `exported/` comes into play.
Types defined here allow for `expected_keepers.go` in other modules to define
contracts that use single canonical types. This pattern allows for code to remain
DRY and also alleviates import cycle chaos.
```